weave/flow/eval.py

```diff
@@ -339,7 +344,12 @@ async def predict_and_score(
                     raise ValueError(
                         f"{score_fn} expects arguments: {score_arg_names}, provide a preprocess_model_input function that returns a dict with those keys."
                     )
-            score_args["output"] = model_output
+            if "model_output" in score_arg_names:
+                util.warn_once(
+                    logger, "model_output is deprecated, please use output instead"
+                )
+                self._output_key = "model_output"
```

Comment:
what happens if some scores use the new output and some use the old? I don't think this should be stored on the eval. just return `output` from predict and score

Reply:
are you saying for all scorers to return `output` instead of `model_output`?

Reply:
i'm saying that `self._output_key` does not make sense. The output key for the arg is per-scorer, not global to the eval

Reply:
I think it does make sense to keep the output key consistent per eval though.  Otherwise if you mix and match scorers, should you get both `output` and `model_output` keys?  That doesn't seem right to me.

Reply:
i think it is worth being clear here - all we are talking about is the key for the output of predict and score (i think?). Which if you miz am match is going to flip around given current implementation. the call stack is 

* predict_and_score
  * predict
  * score 1
  * score 2
  * score 3

the output of `predict_and_score` should not be keyed based on the last scorer param